contentcuration/contentcuration/viewsets/base.py

```diff
@@ -345,12 +347,50 @@ def create(self, validated_data):
         return created_objects
 
 
+class ValuesViewsetOrderingFilter(OrderingFilter):
+
+    def get_default_valid_fields(self, queryset, view, context={}):
```

Comment:
This appears to be overriding the parents method? Maybe add some comments like the sibling method?

Reply:
Can do - and if anything this is probably the more complex change here.